modules/images-imagestream-import-import-mode.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+// * openshift_images/image-streams-managing.adoc
+
+:_content-type: PROCEDURE
+[id="images-imagestream-import-import-mode_{context}"]
+= Setting the import mode when importing images
+
+To set the import mode when importing images, you can use the `--import-mode` flag. This allows you to import all manifests of a manifest list when using `oc import-image` or `oc tag` CLI commands.
+
+.Procedure
+
+* Entering the following command creates an `ImageStreamImport` that includes multi-architecture images, and sets the import mode to `PreserveOriginal`. This ensures that the manifest list and all of its sub-manifests are imported. For example:
++
+[source,terminal]
+----
+$ oc import-image <multiarch-image-stream-tag>  --from=<registry>/<username>/<image-name> \
+--import-mode='PreserveOriginal' --reference-policy=local --confirm
+----
++
+Alternatively, you can import an image with the `Legacy` import mode. This discards manifest lists and imports a single sub-manifest. For example:
++
+[source,terminal]
+----
+$ oc import-image <multiarch-image-stream-tag>  --from=<registry>/<username>/<image-name> \
+--import-mode='Legacy' --confirm
+----
++
+[NOTE]
+====
+The `--import-mode=` default value is `Legacy`. Excluding this value, or failing specify either `Legacy` or `PreserveOriginal`, imports a single sub-manifest.
+
+An invalid import mode returns the following error: `The ImageStreamImport "empty" is invalid: spec.images[0].importPolicy.importMode: Invalid value: "test": invalid import mode, valid modes are '', 'Legacy', 'PreserveOriginal'`.
+====
+
+* Optional: Set the `ImageStream` to periodically update:
++
+[source,terminal]
+----
+$ oc import-image <multiarch-image-stream-tag>  --from=<registry>/<username>/<image-name> \
+--import-mode='PreserveOriginal' --scheduled=true
+----
+
+* Optional: You can set `--insecure=true` so that creating an `ImageStreamImport` skips SSL/TSL verification. For example:
++
+[source,terminal]
+----
+$ oc import-image <multiarch-image-stream-tag> --from=<registry>/<username>/<image-name> \
+--import-mode='PreserveOriginal' --insecure=true
```

Comment:
Does it really make sense to document both `--insecure` and `--scheduled` flags under the `Setting the import mode when importing images` section?
It feels like this becomes more about `oc import-image` than setting import mode specifically. And it's great that we want to improve the `oc import-image` docs, but I think then we need a section about `oc import-image`, where we can talk about `--import-mode`, `--insecure` and `--scheduled`. And perhaps  [`Importing images and image streams from private registries`](https://docs.openshift.com/container-platform/4.12/openshift_images/image-streams-manage.html#images-imagestream-import-images-private-registry_image-streams-managing)  should also be under that new section?

And now I'm going a bit off-topic, but with 4.13 we support specifying import modes on the `oc import-image` command, we could probably remove the [docs about `ImageStreamImport`](https://docs.openshift.com/container-platform/4.12/openshift_images/image-streams-manage.html#importing-manifest-list-through-imagestreamimport_image-streams-managing), as they were added only because it was the only way to specify the import mode back in 4.12.

Reply:
I can make the changes by EOD, and ping you for a review on Monday. 

Thanks for the review @flavianmissi